hack/verify-typecheck.sh

```diff
@@ -28,13 +28,32 @@ kube::golang::verify_go_version
 
 cd "${KUBE_ROOT}"
 
+ret=0
+TYPECHECK_SERIAL="${TYPECHECK_SERIAL:-false}"
+
+SERVER_PLATFORMS=$(echo ${KUBE_SUPPORTED_SERVER_PLATFORMS[@]} | tr ' ' ',')
+CLIENT_PLATFORMS=$(echo ${KUBE_SUPPORTED_CLIENT_PLATFORMS[@]} | tr ' ' ',')
+NODE_PLATFORMS=$(echo ${KUBE_SUPPORTED_NODE_PLATFORMS[@]} | tr ' ' ',')
+TEST_PLATFORMS=$(echo ${KUBE_SUPPORTED_TEST_PLATFORMS[@]} | tr ' ' ',')
```

Comment:
```suggestion
SERVER_PLATFORMS=$(echo "${KUBE_SUPPORTED_SERVER_PLATFORMS[@]}" | tr ' ' ',')
CLIENT_PLATFORMS=$(echo "${KUBE_SUPPORTED_CLIENT_PLATFORMS[@]}" | tr ' ' ',')
NODE_PLATFORMS=$(echo "${KUBE_SUPPORTED_NODE_PLATFORMS[@]}" | tr ' ' ',')
TEST_PLATFORMS=$(echo "${KUBE_SUPPORTED_TEST_PLATFORMS[@]}" | tr ' ' ',')
```

(and below for `${KUBE_SERVER_TARGETS[@]}`)